packages/react-components/src/components/Attachment/AttachmentCardGroup.tsx

```diff
@@ -4,48 +4,56 @@
 import { mergeStyles, Stack } from '@fluentui/react';
 import { _pxToRem } from '@internal/acs-ui-common';
 import React from 'react';
+import { _ATTACHMENT_CARD_MARGIN_IN_PX, _ATTACHMENT_CARD_WIDTH_IN_REM } from '../styles/AttachmentCard.styles';
+import { useattachmentCardGroupLayout } from '../styles/AttachmentCardGroup.styles';
 
 /**
  * @internal
```

Comment:
Should it be private instead of internal?

Reply:
i think internal makes more sense since they are used in couple of different files? It should be private if it's scoped to this file only